src/Attribute/DataSubjectId.php

```diff
@@ -9,4 +9,8 @@
 #[Attribute(Attribute::TARGET_PROPERTY)]
 final class DataSubjectId
 {
+    public function __construct(
+        public readonly string $identifier = 'default',
```

Comment:
I find the naming confusing; `DataSubjectId` has a property called `identifier`. I would use something else like `name`, `group`, or  (for me the best)`scope`.